lib/matplotlib/axes/_axes.py

```diff
@@ -1091,55 +1091,72 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
                   linelengths=1, linewidths=None, colors=None,
                   linestyles='solid', **kwargs):
         """
-        Plot identical parallel lines at specific positions.
+        Plot identical parallel lines at the given positions.
 
-        Plot parallel lines at the given positions.  positions should be a 1D
-        or 2D array-like object, with each row corresponding to a row or column
-        of lines.
+        *positions* should be a 1D or 2D array-like object, with each row
```

Comment:
Currently still using `*`. I will change it if a consensus emerges that it is not the proper markup to use (i.e. if the dev guidelines are updated :) ).